Railway. (patch) Fixes after review

sayam-nasir opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  - SetVariable now returns outputs: projectId, environmentId, serviceId, variableName, variableValue.
  - CreateEnvironment now requires projectId and name.

* **Documentation**
  - Clarified descriptions/tooltips across CreateProject, CreateService, DeployService, FindVariables, and DeleteService (including undeploy behavior and removal of “optional” wording).
  - DeployService output option for Deployment Id removed.

* **Style**
  - Updated icon for GetService.

* **Chores**
  - Bumped Railway bundle version to 1.0.1.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

sayam-nasir committed:
updates

sayam-nasir committed:
Update bundle.json

ZbynekPelunek reviewed (commented):
I forgot to mention in my testing feedback, that I also wasn't able to successfully run **Deploy Service** component, when I provided existing _Service ID_ and _Environment ID_, I got an error:
<img width="948" height="369" alt="Railwau-DeployServiceError" src="https://github.com/user-attachments/assets/4b60ef6e-bb60-4854-98b7-6f84eb7a375f" />

Could you please look into that?

sayam-nasir committed:
updates

sayam-nasir commented:
@ZbynekPelunek I have fixed that now, not sure how it got away, I have now updated the mutation.

rest I have addressed other concerns as well, please check, thanks!
